tasks/update-infra-deployments.yaml

```diff
@@ -38,14 +41,17 @@ spec:
 
         REPO_NAME=$(echo $(params.ORIGIN_REPO) | cut -f5 -d/)
 
-        git clone git@github.com:redhat-appstudio/infra-deployments.git
+        git clone "git@github.com:${TARGET_GH_REPO}" infra-deployments
         cd infra-deployments
         git checkout -b $REPO_NAME
 
         $(params.SCRIPT)
 
         git commit -a -m "$REPO_NAME update" -m "Automatic update"
```

Comment:
Maybe an idea for a future PR, but maybe we could link to the change that generated this PR in the commit message.